specification/dimensions/billingperiodstart.md

```diff
@@ -0,0 +1,32 @@
+# Billing Period Start
+
+Billing period represents the time window for which you have or will receive an invoice for. Billing period represents the time window for which you have or will receive an invoice for. The time window is inclusive of the start date and exclusive of the end date.
```

Comment:
```suggestion
Billing period represents the time window for which an organization have or will receive an invoice for. The time window is inclusive of the start date and exclusive of the end date.
```

Reply:
Given we're using US English spelling, we should probably use US English grammar and change "organization have" to "organization has".